tools/src/bin/collect.rs

```diff
@@ -295,19 +296,30 @@ struct Options {
         long,
         value_parser = HpkeConfigValueParser::new(),
         help_heading = "DAP Task Parameters",
-        display_order = 2
+        display_order = 2,
+        conflicts_with = "hpke_config_json"
     )]
-    hpke_config: HpkeConfig,
+    hpke_config: Option<HpkeConfig>,
     /// The collector's HPKE private key, encoded with base64url
     #[clap(
         long,
         value_parser = PrivateKeyValueParser::new(),
         env,
         help_heading = "DAP Task Parameters",
-        display_order = 3
+        display_order = 3,
+        conflicts_with = "hpke_config_json"
```

Comment:
We should make the two old arguments require each other
```suggestion
        conflicts_with = "hpke_config_json"
        requires = "hpke_config"
```